### DISCUSSION TITLE
how to get callback when rewarded ad is successfully completed or closed in between

### DISCUSSION THREAD

**Discussion by ragarwalpcpl** [admob-plus/admob-plus#501] [Q&A]
rewarded.reward is given in the documentation but its not present in the codes and is undefined when tried in react-native code..

So any idea how to get the callback if reward successful or failed??

Thanks in advance.. please help

**Comment by sksk008** [comment]
i have code of admob free in ionic i think its can help you
```
this.adssubcriptionra = this.admobFree.on('admob.rewardvideo.events.CLOSE').subscribe(() => {
			console.log('rewardVideoclose')
		});
```
and in admob cordova it is 
"admob.rewarded.dismiss" for more [click here ](https://admob-plus.github.io/docs/cordova/api/enums/Events)
